21_node_management/22_search_nodes.txt

```diff
@@ -10,17 +10,76 @@ about the Nodes which have been already validated, and are managed by Rudder.
 
 ==== Quick Search
 
-The easiest search tool is the Quick search: type in the search field the first
-letters of the 'Rudder ID', 'Reference', or 'Hostname'; choose the accurate Node in
-the autocompletion list; validate and look at the Node information. This search
-tool can be very useful to help you create a new search in the Advanced Search.
-
-.Quick search the Node called +debian-node+
+You might have noticed the small text area at the top of the Rudder interface:
+it is the Quick Search bar. Its purpose is to enable a user to easily search for
+Rudder elements (Nodes, Groups, Directives, Parameters, Rules)  based on their
+name, id, description, inventory...
+
+image::./images/rudder-quicksearch.png[Quicksearch]
+
+An autocompletion list will appear as soon as Rudder detects an element it can
+identify, you just have to click on it to be redirected to the element
+configuration page.
+
+More complex search queries can be input using the "in:" and "is:" keywords,
+"in" targetting a Rudder category and "is" targetting elements like name,
+description...
+
+Those keywords are used to refine a research in case an entry returns too many
+element.
+
+The available search keywords are:
+
+.in: keywords
+[frame="topbot",options="header"]
+|======================
+|keyword |Description
+|node      |Nodes
+|group     |Groups
+|directive |Directives
+|parameter |Parameters
+|rule      |Rules
+|======================
+
+.is: keywords
+[frame="topbot",options="header"]
+|======================
```

Comment:
Mostly you only added the 'aliases' we defined and not the main attribute name we defined